project.clj

```diff
@@ -8,7 +8,7 @@
                         ["snapshots" {:url "https://clojars.org/repo" :creds :gpg}]]
   :dependencies [[org.clojure/clojure "1.9.0"]
                  [org.clojure/tools.logging "0.4.1"]
-                 [threatgrid/turnstile "0.102"]
+                 [threatgrid/turnstile "0.103-SNAPSHOT"]
```

Comment:
just a comment to remind you to deploy a non snapshot version before merging it.